manifest.go

```diff
@@ -171,10 +196,11 @@ func readManifest(r io.Reader) (*Manifest, []error, error) {
 
 func fromRawManifest(raw rawManifest) (*Manifest, error) {
 	m := &Manifest{
-		Constraints: make(gps.ProjectConstraints, len(raw.Constraints)),
-		Ovr:         make(gps.ProjectConstraints, len(raw.Overrides)),
-		Ignored:     raw.Ignored,
-		Required:    raw.Required,
+		Constraints:  make(gps.ProjectConstraints, len(raw.Constraints)),
+		Ovr:          make(gps.ProjectConstraints, len(raw.Overrides)),
+		Ignored:      raw.Ignored,
+		Required:     raw.Required,
+		PruneOptions: gps.PruneNestedVendorDirs,
```

Comment:
Good catch. I could change this so that coming through `dep/cmd/dep` always toggle that flag in `PruneOptions` when passing it to `SafeWriter.Write` or `gps.WriteDepTree`.

I understood from @sdboyer that we want `gps` to have control over this flag, but `dep` should always keep it on.